mlflow/metrics/genai/genai_metric.py

```diff
@@ -243,7 +244,15 @@ def score_model_on_one_payload(
             eval_parameters,
             eval_model,
         ):
-            arg_string = _format_args_string(grading_context_columns, eval_values, indx)
+            try:
+                arg_string = _format_args_string(grading_context_columns, eval_values, indx)
+            except Exception as e:
+                raise MlflowException(
+                    f"Values for grading_context_columns are malformed and cannot be "
+                    f"formatted into prompt for metric '{name}'.\n"
+                    f"Provided values: {eval_values}\n"
```

Comment:
The error message shown in the screenshot (in PR description) is a bit confusing - it says `grading_context_columns` is incorrectly set but the value shows `{'targets': None}`. I cannot link `targets` to `grading_context_columns`, so I don't know how I can fix the error.